implementations/rust/ockam/ockam_api/src/nodes/models/transport.rs

```diff
@@ -176,6 +177,10 @@ impl<'a> TransportStatus<'a> {
             tid: tid.into(),
         }
     }
+
+    pub fn socket_addr(&self) -> std::result::Result<SocketAddrV4, AddrParseError> {
```

Comment:
This should return an `ockam_core::Result`, manually mapping the `AddrParseError` into an `ockam_core::Error`.